plugins/amazonq/codewhisperer/jetbrains-community/tst/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeFileScanTest.kt

```diff
@@ -166,6 +169,131 @@ class CodeWhispererCodeFileScanTest : CodeWhispererCodeScanTestBase(PythonCodeIn
         }
     }
 
+    @Test
+    fun `test run() - measure CPU and memory usage`() {
+        // Create a mock payload file of 198KB in bytes
+        val mockPayloadFile = createMockPayloadFile(198 * 1024)
+
+        // Set up the mock payload
+        val mockPayloadContext = PayloadContext(
+            CodewhispererLanguage.Python,
+            1,
+            1,
+            10,
+            listOf(),
+            600,
+            199 * 1024
```

Comment:
set up this context creation in a function instead of using constants directly like this.